### DISCUSSION TITLE
执行工作流后，怎么获取对应的任务实例id？

### DISCUSSION THREAD

**Discussion by tangjiaolong** [apache/dolphinscheduler#12224] [Q&A]
因为工作流可能存在多次频繁执行，就会创建多个任务实例，我怎么找到本次执行的工作流对应的任务实例id呢？

**Comment by DarkAssassinator** [comment]
please describe your question in English.

**Comment by DarkAssassinator** [comment]
u can check the db `t_ds_task_instance` table, `process_instance_id` is workflow instance id.
